server/enterprise/metrics/metrics.go

```diff
@@ -1573,6 +1579,42 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.DesktopClientMemoryUsage)
 
+	m.AccessControlEngineInitDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
```

Comment:
Add `ConstLabels: additionalLabels` to every metric. This will be refactored to a common initializer.